packages/core/core/src/resolveOptions.js

```diff
@@ -50,7 +50,19 @@ export default async function resolveOptions(
     entries = [path.resolve(inputCwd, initialOptions.entries)];
   }
 
-  let entryRoot = getRootDir(entries);
+  let entryRoot = getRootDir(
+    await Promise.all(
+      entries.map(async entry =>
+        // if the entry refers entryRootDir, getRootDir returns the parent directory of entryRootDir.
+        // In this case, each entry should refers some file, but without cli build target entry refers entryRootDir directory.
+        // So, we resolve with index as entry to make entry refers some (virtual) file in such case.
```

Comment:
```suggestion
        // getRootDir treats the input as files, so getRootDir(["/home/user/myproject"]) returns "/home/user".
        // Instead we need to make the entries refer to some file inside the specified folders.
```
I was having trouble understanding your comment 😄

Reply:
Looks good ☺️ The comment you changed is easier to understand! Thank you!